frontend/lib/src/components/shared/Radio/Radio.tsx

```diff
@@ -165,7 +168,28 @@ class Radio extends React.PureComponent<Props, State> {
                 allowHTML={false}
                 isLabel
                 isButton
+                largerLabel
               />
+              {captions[index] && (
+                <StreamlitMarkdown
+                  source={captions[index]}
+                  allowHTML={false}
+                  isLabel
+                  isButton
```

Comment:
Based on the description, `isButton` is used here to disable links. But why do we need to do that? Using links in these captions seems to be a reasonable and useful usage?  Also, if we reuse this `isButton` flag across different non-button components, maybe we can rename to something like `disableLinks`.

Reply:
That makes sense, re-enable links just in the captions and renamed `isButton` flag to `disableLinks`